src/Microsoft.Health.Fhir.Liquid.Converter.FunctionalTests/TestData/Expected/Hl7v2/ADT_A01/ADT01-23-expected.json

```diff
@@ -3,10 +3,40 @@
   "type": "transaction",
   "entry": [
     {
-      "fullUrl": "urn:uuid:64bac34e-e611-3549-848b-89416176aa0b",
+      "fullUrl": "urn:uuid:",
```

Comment:
Can we keep ID in the groundtruth?

Reply:
Sure, and I'll revert the latest commit since it will have conflict with another branch. In the following commits I'll add the ID.